### PR TITLE
Check for things to clean before trying

### PR DESCRIPTION
**Pull request by dan9186**
PR for #27 to be smarter about the `docker clean` command internals.

### CODE REVIEW

**Comment by brimstone** [comment]
It'd be cooler if this only hit `docker ps -a -q -f "status=exited"` once, then looped through the result, if any. If @dan9186 doesn't want to do this, or @itsamenathan puts up a fight, this is fine with me.

**Comment by itsamenathan** [comment]
I'm not going to fight that.

File: modules/docker/docker [hunk 1]
```diff
@@ -14,7 +14,10 @@ function docker() {
 	clean)
 		case "$mod" in
 		containers)
-			command docker rm -v $(command docker ps -a -q -f status=exited) || return 0
+			local exits=$(docker ps -a -q -f "status=exited")
```
Comment (brimstone) [inline modules/docker/docker]: There isn't a need for double quotes around the `$()`?

**Comment by brimstone** [comment]
LGTM

**Pull request merged**